src/index.ts

```diff
@@ -308,7 +345,19 @@ const SplunkRum: SplunkOtelWebType = {
     }
 
     _errorInstrumentation.report('SplunkRum.error', args);
-  }
+  },
+
+  _experimental_addEventListener(name: SplunkOtelWebEventType, callback: (event: never) => void): void {
```

Comment:
What's the point of having internal event target methods also have experimental prefix when it's publicly only exposed here? Could get rid of the custom `NativeEventTarget`

Reply:
It's just convenient to reuse `SplunkOtelWebEventTarget` interface internally and externally.